packages/plugins/plugin-device-token/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@segment/analytics-react-native-plugin-device-token",
+  "version": "0.1.0",
+  "description": "The hassle-free way to add Segment analytics to your React-Native app.",
+  "main": "lib/commonjs/index",
+  "scripts": {
+    "build": "bob build",
+    "test": "jest",
+    "typescript": "tsc --noEmit",
+    "clean": "rimraf lib node_modules",
+    "release": "semantic-release"
+  },
+  "keywords": [
+    "segment",
+    "react-native",
+    "ios",
+    "android"
+  ],
+  "module": "lib/module/index",
+  "types": "lib/typescript/src/index.d.ts",
+  "react-native": "src/index",
+  "source": "src/index",
+  "files": [
+    "src",
+    "lib",
+    "android",
+    "ios",
+    "cpp",
+    "segment-analytics-react-native.podspec",
+    "package.json",
+    "!src/**/*.e2e.mock.js",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/segmentio/analytics-react-native.git",
+    "directory": "packages/core"
+  },
+  "author": "Segment <hello@segment.com> (https://segment.com/)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/segmentio/analytics-react-native/issues"
+  },
+  "homepage": "https://github.com/segmentio/analytics-react-native/tree/master/packages/plugins/plugin-firebase#readme",
+  "peerDependencies": {
+    "@react-native-firebase/app": "17.3.2",
```

Comment:
it shouldn't be such an specific version, I think `17.x` is fine, we only need to stick it to a major version.